lib/LedgerSMB/Template/DB.pm

```diff
@@ -141,8 +141,11 @@ sub get_from_file {
     my $content = '';
     open my $fh, '<', $path
         or die "Failed to open template file $path : $!";
-    $content .= $_ while <$fh>;
-    close $fh;
+    {
+        local $/ = undef;
+        $content =  <$fh>;
+    }
+    close $fh or die "Cannot close file $path";
```

Comment:
@sbts the block changes a large(ish) number of (line) reads to a single line read by setting the EOL character variable to `undef`. It's roughly the UTF-8 compliant variant of slurping a file into memory.